DEVELOPMENT.adoc

```diff
@@ -0,0 +1,75 @@
+# oVirt Engine - Dev Container setup & building
+
+Before beginning, ensure that [Docker](https://www.docker.com/) is installed on the machine that the Dev Container will be running on. This could be your local device or on a [remote machine](https://code.visualstudio.com/docs/devcontainers/containers#_open-a-folder-on-a-remote-ssh-host-in-a-container).
+
+### Setup the PostgresQL database
+
+To begin, navigate to the project directory. If working remotely, SSH into your remote environment and locate your source files. To fully set up your PostgreSQL container with testing credentials, execute the Docker Compose file by running:
+
+```
+docker compose up -d
+```
+
+The `-d` flag runs the container in detached mode. 
+
+> It's crucial to run the compose file prior to creating the development container; otherwise, Docker networking problems may arise.
+
+### Connect to your Dev Container and build the application
+
+#### Open folder in a Dev Container
+
+When you are running the container on your own device, open the project directory in Visual Studio Code, and use `Dev Containers: Reopen in Container` command from the Command Palette (F1). 
+
+For remote connections, follow the same steps, but start by connecting to your remote environment first. Utilize the Remote Explorer feature in VSC to SSH into a remote server. You'll find this in the 'Remote Explorer' tab. Just make sure you've installed the "Remote - SSH" and "Remote Explorer" extensions from Microsoft for this to work.
+
+More information on Dev Containers can be found [here](https://code.visualstudio.com/docs/devcontainers/containers).
+
+### Building the application
+
+You can set up the development environment by executing the following command:
+
+```
+make install-dev PREFIX=/home/build/ovirt/
+```
+
+Running this command will initiate the application's build process. You can skip tests by appending the `SKIP_TESTS=1` flag to the previous command. Additionally, you can specify the installation directory using the `PREFIX` flag.
```

Comment:
env vars are always set at the beginning of the command normally :)
SO: `SKIP_TESTS=1 make install-dev PREFIX=/home/build/ovirt/`